kolibri_instant_schools_plugin/instant_schools_settings.py

```diff
@@ -1,25 +1,10 @@
 # -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-from kolibri.deployment.default.settings.base import *
+INSTALLED_APPS = ["django.contrib.admin"]
```

Comment:
Why do we not have to append the module to INSTALLED_APPS here, but instead just define it as an array fresh?

Reply:
It works this way to avoid plugins potentially interfering with each other - the Kolibri plugin machinery grabs any installed apps defined like this and appends them to the existing installed apps.

If we required them each to import the installed apps from somewhere, they might end up fighting.